applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -78,6 +94,17 @@ public static ConfigurationAsyncClientBuilder builder() {
      * Adds a configuration value in the service if that key and label does not exist. The label value of the
      * ConfigurationSetting is optional.
      *
+     * <p><strong>Code Samples</strong></p>
+     * <pre>
+     * ConfigurationSetting setting = new ConfigurationSetting().key("prodDBConnection")
+     *     .label("westUS")
+     *     .value("db_connection");
+     *
+     * client.addSetting(setting).subscribe(response -&gt; {
```

Comment:
I would prefer to see this kind of code sample with the ConfigurationSetting passed directly into addSetting, without the intermediate `setting` variable. Ping me to get a good style.